shared/utils/combine-remediations.py

```diff
@@ -423,21 +423,21 @@ def main():
                         else:
                             mod_file += line
 
-                complexity = None
-                disruption = None
-                reboot = None
+                complexity = "unknown"
+                disruption = "unknown"
+                reboot = "false"
```

Comment:
@mpreisler the defaults are based off of the specification which says that the default is '0'.